python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -414,7 +414,9 @@ def get(self):
                     'mine_name': x.mine_name,
                     'mine_no': x.mine_no,
                     'latitude': str(x.mine_location.latitude) if x.mine_location else '',
-                    'longitude': str(x.mine_location.longitude) if x.mine_location else '', },
+                    'longitude': str(x.mine_location.longitude) if x.mine_location else '',
+                    'mine_location_description': x.mine_location.mine_location_description if x.mine_location else '',
+                    },
```

Comment:
This resource should have been updated to use the new marshalling, which removes the need for these type coercions + null checks. I'm not sure why it wasn't updated with the other resources in this file. I think it needs to be prioritized ASAP unless it was intentionally skipped. @JustinMacaulay ?

I'm concerned that we're returning an empty string for lat/long instead of `None`/`null`. That will make switching to marshalling either unnecessarily difficult, a breaking API change (switch to returning `null`), or a db update to make lat/lon default to an empty string (which seems like the worst option to me).